src/oss-tests/Properties/Resources.Designer.cs

```diff
@@ -61,7 +61,7 @@ internal class Resources {
         }
         
         /// <summary>
-        ///   Looks up a localized string similar to {&quot;_id&quot;:&quot;@angular/core&quot;,&quot;_rev&quot;:&quot;657-0ac278fe941aa315fa5b063b2d70170d&quot;,&quot;name&quot;:&quot;@angular/core&quot;,&quot;dist-tags&quot;:{&quot;latest&quot;:&quot;13.2.5&quot;,&quot;v4-lts&quot;:&quot;4.4.7&quot;,&quot;v5-lts&quot;:&quot;5.2.11&quot;,&quot;next&quot;:&quot;14.0.0-next.5&quot;,&quot;v7-lts&quot;:&quot;7.2.15&quot;,&quot;v6-lts&quot;:&quot;6.1.10&quot;,&quot;v9-lts&quot;:&quot;9.1.13&quot;,&quot;v8-lts&quot;:&quot;8.2.14&quot;,&quot;v10-lts&quot;:&quot;10.2.5&quot;,&quot;v11-lts&quot;:&quot;11.2.14&quot;,&quot;v12-lts&quot;:&quot;12.2.16&quot;},&quot;versions&quot;:{&quot;0.0.0-0&quot;:{&quot;name&quot;:&quot;@angular/core&quot;,&quot;version&quot;:&quot;0.0.0-0&quot;,&quot;description&quot;:&quot;&quot;,&quot;main&quot;:&quot;index.js&quot;,&quot;jsnext:main&quot;:&quot;esm/index.js&quot;,&quot;typings&quot;:&quot;index.d.ts&quot;,&quot;author&quot;:{&quot;name&quot;:&quot;angular&quot;},&quot;license&quot;:&quot;MIT&quot;,&quot;p [rest of string was tru....
+        ///   Looks up a localized string similar to {&quot;_id&quot;:&quot;@angular/core&quot;,&quot;_rev&quot;:&quot;660-3fe4d5b1b9e632c7d96cdc1e1005b13c&quot;,&quot;name&quot;:&quot;@angular/core&quot;,&quot;dist-tags&quot;:{&quot;latest&quot;:&quot;13.2.6&quot;,&quot;v4-lts&quot;:&quot;4.4.7&quot;,&quot;v5-lts&quot;:&quot;5.2.11&quot;,&quot;next&quot;:&quot;13.3.0-rc.0&quot;,&quot;v7-lts&quot;:&quot;7.2.15&quot;,&quot;v6-lts&quot;:&quot;6.1.10&quot;,&quot;v9-lts&quot;:&quot;9.1.13&quot;,&quot;v8-lts&quot;:&quot;8.2.14&quot;,&quot;v10-lts&quot;:&quot;10.2.5&quot;,&quot;v11-lts&quot;:&quot;11.2.14&quot;,&quot;v12-lts&quot;:&quot;12.2.16&quot;},&quot;versions&quot;:{&quot;0.0.0-0&quot;:{&quot;name&quot;:&quot;@angular/core&quot;,&quot;version&quot;:&quot;0.0.0-0&quot;,&quot;description&quot;:&quot;&quot;,&quot;main&quot;:&quot;index.js&quot;,&quot;jsnext:main&quot;:&quot;esm/index.js&quot;,&quot;typings&quot;:&quot;index.d.ts&quot;,&quot;author&quot;:{&quot;name&quot;:&quot;angular&quot;},&quot;license&quot;:&quot;MIT&quot;,&quot;pee [rest of string was tru....
```

Comment:
Where is the resources.resx file that generated this?

Reply:
I was trying to find it, but I think it's because the resources.resx didn't change. The json files already existed, and the changes are due to that I changed the actual json file contents, so the Reources.Designer changed to reflect the contents of the json files changing.

Reply:
Ah so the compilation extracts the contents of the referenced json and embeds it here?

Reply:
Pretty sure that's what it does, yes.

Reply:
Since this file is generated should it even be checked in? or should it be gitignored?

Reply:
I don't think it gets auto generated. From the research I did, it needs to be checked in as well. https://stackoverflow.com/questions/69876372/why-cant-i-delete-designer-cs-files